src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -80,8 +81,11 @@ public void start(Future<Void> startFuture) {
     }
 
     //set Config object's defaults
-    int port = config().getInteger("port");
-    NetServerOptions options = new NetServerOptions().setPort(port);
+    int port = config().getInteger("port"); // move port to netServerOptions
```

Comment:
The benefit is minimal. Basically, we wouldn't need to call `NetServerOptions.setPort()`. The JSON sent in `netServerOptions` is the serialized form of `NetServerOptions`, which includes a `port` field, the one that `setPort()` sets. I didn't want to mess with the existing config, so I left it in place, but it might be less confusing to have `port` in one place. I don't explicitly mention `port` as a `netServerOption` in the README, so I guess it might only be confusing to those who read the linked `NetServerOptions` Javadoc mentioned in the README.